youtube_dl/extractor/techtalks.py

```diff
@@ -10,7 +10,7 @@
 
 
 class TechTalksIE(InfoExtractor):
-    _VALID_URL = r'https?://techtalks\.tv/talks/[^/]*/(?P<id>\d+)/'
+    _VALID_URL = r'https?://techtalks\.tv/talks/(?:[^/]*/?)(?P<id>\d+)/'
```

Comment:
With this you allow URLs like http://techtalks.tv/talks/learning-topic-models-going-beyond-svd57758/ that's incorrect.